data/global.lua

```diff
@@ -2,6 +2,7 @@ math.randomseed(os.time())
 dofile('data/lib/lib.lua')
 
 STORAGEVALUE_PROMOTION = 30018
+DELAY_LARGE_SEASHELL = 30019
```

Comment:
The constant name says nothing about the type, i.e. it should be named `STORAGEVALUE_DELAY_LARGE_SEASHELL`